src/mpl2/src/hier_rtlmp.cpp

```diff
@@ -4068,16 +4072,6 @@ void HierRTLMP::writeMacroPlacement(const std::string& file_name)
 
 void HierRTLMP::clear()
 {
-  for (auto& [module, metrics] : tree_.maps.module_to_metrics) {
-    delete metrics;
-  }
-  tree_.maps.module_to_metrics.clear();
```

Comment:
Do these no longer need to be cleared?

Reply:
They do. I'm fixing it.

Reply:
When HierRTLMP::run() finishes the tree will be deleted and so these maps which should destroy the unique_ptrs so I don't think these clear() are needed. When the Cluster memory management rework (the subsequent PR which will be opened) is merged. This clear() function will no longer exist and the eraseDrawing  will be moved to HierRTLMP::run();

Reply:
That sounds reasonable